### ISSUE TITLE
Add cookies to downloadFile method

### ISSUE THREAD

**Issue by ldikmans** [ionic-team/capacitor-plugins#1893]
### Plugin
Filesystem

### Description
we download a file from our server. This server checks if the user that downloads the file is authorized for this file by checking the session. Download file does not send the session cookie. Hence we can't use downloadFile to download the file 

### Platform(s)
Android, web and iOS

### Preferred Solution
Add the cookie to the request like the nativeHttp plugin does

### Alternatives
I have written the code myself, using a link and using writeFile etc

### Additional Context
n.a.

**Comment by aanders77** [comment]
I agree with TS. Sessions is largely used still, and without the ability to include session cookies this plugin lacks support for many use-cases.
